content-tool/tool/src/webapp/vm/resources/sakai_create_uploads.vm

```diff
@@ -131,8 +131,12 @@
     #if($dragAndDrop)
         <div id="dragDropWrapper">
         <div id="fileUploaderDesc" class="instruction">
-            #if($label_dragDescription)
-                <p>$label_dragDescription</p>
+            <p>$tlang.getString("label.dragDescription")</p>
+            #if($label_overwrite_warning)
+                <div class="warning">
```

Comment:
Sakai has a standard class for doing this of `alertMessage` which most tools uses and is more accessible.